elfi/methods/results.py

```diff
@@ -203,6 +204,36 @@ def sample_means_array(self):
         """
         return np.array(list(self.sample_means.values()))
 
+    def save_samples(self, name=None, type_='csv'):
+        """Save samples in csv or json file formats.
+
+        Parameters
+        ----------
+        name : file name to be saved
+        type_ : file format to be saved
+
+        """
+        import json
+        import csv
+
+        if not name:
+            name = self.__class__.__name__.lower() + '_' + 'samples.' + type_
```

Comment:
This results in 'sample_samples.csv'. Not very good. Then again the user doesn't know where the file goes with what name. Maybe it's better to just give an error?